src/main/drivers/bus.h

```diff
@@ -116,6 +116,7 @@ typedef enum {
     DEVHW_PCA9685,      // PWM output device
     DEVHW_M25P16,       // SPI NOR flash
     DEVHW_UG2864,       // I2C OLED display
+    DEVHW_PITOT_EXP,    // Pitot meter
```

Comment:
What's "EXP"? If this is an airspeed sensor from Eagle Tree Vector it would be better to call accordingly.

Reply:
The name of the sensor is "PITOT_EXP" so that's what I put, but maybe something like "EAGLE_TREE_VECTOR_PITOT_EXP" might be better?

Reply:
I'm thinking about `PITOT_ETVECTOR` - it's reasonably short and to some degree self-explanatory

Reply:
or "PITOT_EAGLE_TREE_EXP"

Reply:
Here it should short: `DEVHW_EAGLE_TREE_EXP` or `DEVHW_ETEXP`

Reply:
or just "PITOT_EAGLE_TREE"

Reply:
`DEVHW_PITOT_EAGLE_TREE` here - to indicate that it's a device hardware ID